doc/source/serve/advanced-guides/index.md

```diff
@@ -13,6 +13,7 @@ dev-workflow
 grpc-guide
 managing-java-deployments
 deploy-vm
+multi-app-container
 ```
 
 If you’re new to Ray Serve, we recommend starting with the [Ray Serve Quickstart](serve-getting-started).
```

Comment:
Would be nice to add to this list as well so it's also in the page and not just on the side bar